otx/algorithms/detection/utils/otx_utils.py

```diff
@@ -0,0 +1,220 @@
+"""Utils for OTX Detection."""
```

Comment:
Should we keep this just to `data.py` and the `otx_utils.py` to just `utils.py`? As they are in the utils folder the postfix can be dropped.

Reply:
I agree with the comments and renamed this.